.github/workflows/merge.yml

```diff
@@ -179,24 +179,59 @@ jobs:
           SN_LOG: "all"
         timeout-minutes: 2
 
-      - name: Start a client to create a register
+      - name: Start a client to create a register writable only by the owner
         run: cargo run --bin safe --release -- --log-output-dest=data-dir register create -n baobao
         env:
           SN_LOG: "all"
         timeout-minutes: 10
 
-      - name: Start a client to get a register
+      - name: Start a client to get a register writable only by the owner (client is the owner)
         run: cargo run --bin safe --release -- --log-output-dest=data-dir register get -n baobao
         env:
           SN_LOG: "all"
         timeout-minutes: 2
 
-      - name: Start a client to edit a register
+      - name: Start a client to edit a register writable only by the owner (client is the owner)
         run: cargo run --bin safe --release -- --log-output-dest=data-dir register edit -n baobao wood
         env:
           SN_LOG: "all"
         timeout-minutes: 10
 
+      - name: Start a client to create a register writable by anyone
+        run: cargo run --bin safe --release -- --log-output-dest=data-dir register create -n trycatch -p
+        env:
+          SN_LOG: "all"
+        timeout-minutes: 10
+
+      - name: Start a client to get a register writable by anyone (client is the owner)
+        run: cargo run --bin safe --release -- --log-output-dest=data-dir register get -n trycatch
+        env:
+          SN_LOG: "all"
+        timeout-minutes: 2
+
+      - name: Start a client to edit a register writable by anyone (client is the owner)
+        run: cargo run --bin safe --release -- --log-output-dest=data-dir register edit -n trycatch water
+        env:
+          SN_LOG: "all"
+        timeout-minutes: 10
+
+      - name: Delete client subdir to generate new client
+        shell: bash
+        run: rm -rf "${{ matrix.safe_path }}"/client
+
+      - name: Start a client to get a register writable by anyone (new client, the owner of Register)
+        run: cargo run --bin safe --release -- --log-output-dest=data-dir register get -n trycatch
```

Comment:
I'm not sure how to write a successful test case for getting the "public" register.  The problem is that CLI needs the hex address of the public register in this case - it cannot take one specified by name with -n if it was created by a different client.  But there is no way to know that address... unless it is parsed out somehow from the output of the CLI command when it was created above with -p.  I do not know how.

Reply:
Hey @thomasmarkiewicz , thanks for the work so far here!

Sounds like we need this actually printed to std out as part of the PUT process? And then you could save that to a github variable to share between steps?

Do you think it makes sense to _always_ print this info? Or should we hide it behind a flag in the CLI?